docs/tutorial/fdfitting.rst

```diff
@@ -0,0 +1,314 @@
+the inversion is done numerically, rather than analytically. For more complex examples on how this
+inversion may be used, please see the examples.
+
+For a full list of models that are available, please refer to the documentation by invoking
+`help(pylake.fitting.models)`.
+
+
+Loading data
+------------
+
+Next up, is loading some data. Let's assume we have two data sets. One was acquired in the presence
+of a ligand, and another was measured without a ligand. We expect this ligand to only affect the 
+contour length of our DNA. Loading the first dataset is simple. Note that we explicitly define
+which dataset is force and which is distance via keyed arguments (this is intended to prevent
+mistakes in argument order)::
+
+    data1 = odijk_with_offsets.load_data(f=force1, d=distance1, name="Control")
+
+Note how load_data returns a handle to the loaded data. We store this in `data1`. Such handles 
+contain information about which parameters are used in that simulation condition. They can be
+used to get more fine grained control over what is plotted or simulated. More on that later in
+this tutorial. For the second dataset, we want the contour length to be different. We can achieve
+this by renaming it when loading the data::
+
+    data2 = odijk_with_offsets.load_data(f=force2, d=distance2, name="RecA", DNA_Lc="DNA_Lc_RecA")
+
+More specifically, we renamed the parameter `DNA_Lc` to `DNA_Lc_RecA`. Sometimes, you may want
+a large number of datasets with different offsets. Assuming we have two lists of distance and
+force vectors stored in the lists distances and forces. In this case, it may make sense to load
+them in a loop and set such transformations programmatically::
+
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        odijk_with_offsets.load_data(f=force, d=distance, name="RecA", f_offset=f"f_offset_{i}")
+
+The syntax `f"offset_{i}"` is parsed into `offset_0`, `offset_1` ... etc. For more information
+on how this works, read up on Python fantastic f-Strings.
+
+Fitting the data
+----------------
+
+Once the data loaded, we can fit the data. To do this, we have to create a `FitObject`. This 
+object will collect all the parameters involved in the models and data, and will allow you to 
+interact with the model parameters and fit them. We construct it using `pylake.FitObject` and 
+passing it one or more models. In return, we get an object we can interact with, which in this
+case we store in `odijk_fit`::
+
+    odijk_fit = pylake.FitObject(odijk_with_offsets)
+
+The parameters of the model can be accessed under `parameters`. Note that by default, parameters 
+tend to have reasonable initial guesses and bounds in pylake, but we can set our initial guess and
+a lower and upper bound as follows::
+
+    odijk_fit.parameters["DNA_Lp"].value = 50
+    odijk_fit.parameters["DNA_Lp"].lb = 39
+    odijk_fit.parameters["DNA_Lp"].ub = 80
```

Comment:
Parameters seem to be the main use of this class. Can we just give it a `__getitem__` to simplify things?
```suggestion
    odijk_fit["DNA_Lp"].value = 50
    odijk_fit["DNA_Lp"].lb = 39
    odijk_fit["DNA_Lp"].ub = 80
```

Reply:
Yep, definitely more beautiful. 👍